src/portal/src/app/project/repository/artifact-list-page/artifact-list/artifact-list-tab/artifact-list-tab.component.spec.ts

```diff
@@ -359,17 +358,16 @@ describe("ArtifactListTabComponent (inline template)", () => {
     fixture.detectChanges();
   });
 
-  it("should load and render data", () => {
+  it("should load and render data", async () => {
```

Comment:
waitForAsync ？

Reply:
This is  "async  and await", "waitForAsync()" replaces "async()"